Makefile.am

```diff
@@ -53,15 +59,21 @@ stream_DATA = $(NULL)
 recordingsdir = $(datadir)/janus/recordings
 recordings_DATA = $(NULL)
 
+if WINDOWS_OS
+INSTDIR = ../..
```

Comment:
Why this? Shouldn't this be decided by the prefix you pass when configuring? Or does this work differently with MingW on Windows?